docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -54,6 +54,25 @@ you'll need to push another commit after correcting it to activate the sync to D
     - Are all indentation levels correct?
     - Does the name field match, if applicable?
 
+The changes made to my GitHub repo aren't appearing on Dockstore, but I've already installed the GitHub app and made the ``.dockstore.yml`` file. How can I figure out what's going wrong?
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+If you've already tried waiting a couple of minutes and refreshing the browser on the My Workflows page, you can view GitHub App logs through Dockstore to see if there have been any errors.
+Navigate to the ``/my-workflows`` page and expand the GitHub Organization that the repository belongs to on the left hand side. Then click on the bottom where it says ``See GitHub App Logs``.
+
+.. image:: /assets/images/docs/github-app-logs-button.png
+   :width: 40 %
+
+Once loaded, the following window will be displayed.
+
+.. image:: /assets/images/docs/github-app-logs-window.png
+
+Here you can view all the GitHub app events Dockstore is aware of and whether they failed or were successful. If there was a failure, you can expand that row and view the error message as shown below.
```

Comment:
It would nice if you explained the error, e.g., "in this case the .dockstore.yml had a key with the value `test` that is not part of the dockstore.yml schema` (could be worded better).

Reply:
explained